pkg/instrumentation/defaultinstrumentation.go

```diff
@@ -14,10 +14,11 @@ import (
 )
 
 const (
-	defaultAPIVersion                      = "cloudwatch.aws.amazon.com/v1alpha1"
-	defaultInstrumenation                  = "java-instrumentation"
-	defaultNamespace                       = "default"
-	defaultKind                            = "Instrumentation"
+	defaultAPIVersion     = "cloudwatch.aws.amazon.com/v1alpha1"
+	defaultInstrumenation = "java-instrumentation"
+	defaultNamespace      = "default"
+	defaultKind           = "Instrumentation"
```

Comment:
Does this have any code change? Or we are just reformatting the code?

Reply:
What does default instrumentation and config now mean when we have multiple instrumentations? I don't think OTel operator has a default instrumentation. It will inject whatever instrumentation is defined via the annotation.

Reply:
the Instrumentation CR defines multiple languages. The default instrumentation will mutate pods even if the customer has no Instrumentation CR on their cluster.

This file has the definition of what the environment variables we will inject when pods are annotated.

Reply:
>The default instrumentation will mutate pods even if the customer has no Instrumentation CR on their cluster.

Still not clear why this is needed to be done.